src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import './Footer.css'
+
+export const Footer = () => {
+  return (
+    <div>
+      <footer className="footer">
+        <div className="footer-header dis_flex">
+          <p>Made by Tanjum</p>
+        </div>
+        <ul className="social_media_links dis_flex gap_s">
+          <li className="list-item-inline">
+            <a href="https://github.com/Tanjum786" target="blank_">
```

Comment:
I think it's a typo here for target="_blank"?

Reply:
yeah i will correct it